### PR TITLE
Make sure supplied inputs were wanted, reenable more tests

### PR DESCRIPTION
**Pull request by cjllanwarne**

### CODE REVIEW

File: centaur/src/main/resources/standardTestCases/sub_workflow_decls/sub_workflow_decls.wdl [hunk 1]
```diff
@@ -11,15 +11,13 @@ workflow sub_workflow_decls {
...
-    #, optional_without_default_supplied = "consider yourself supplied"
-    , optional_with_default_but_overridden = "supplied but overridden optional"
-    , passthrough_required_input = workflow_required_input
-    , passthrough_initialized_input = workflow_initialized_input
-    , passthrough_overridden_input = workflow_overridden_input
+    needs_to_be_supplied = "initialized",
+    default_which_is_overridden = "overridden",
+    optional_without_default_supplied = "consider yourself supplied",
+    optional_with_default_but_overridden = "supplied but overridden optional",
+    passthrough_required_input = workflow_required_input,
```
Comment (mcovarr) [inline centaur/src/main/resources/standardTestCases/sub_workflow_decls/sub_workflow_decls.wdl]: goofed up highlighting

File: centaur/src/main/resources/standardTestCases/unexpected_subworkflow_call_input_failure/unexpected_subworkflow_call_input_failure.wdl [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import "unexpected_subworkflow_call_input_failure_import.wdl" as subwf
+
+workflow unexpected_subworkflow_call_input_failure {
+  call subwf.subwf { input: i = 10, j = 20 }
+}
```
Comment (mcovarr) [inline centaur/src/main/resources/standardTestCases/unexpected_subworkflow_call_input_failure/unexpected_subworkflow_call_input_failure.wdl]: \n here and below

File: centaur/src/main/resources/standardTestCases/public_http_import/public_http_import.wdl [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import "https://raw.githubusercontent.com/broadinstitute/cromwell/f2d1c3bdd5535d9f6a997eadaf136742d86adbe5/centaur/src/main/resources/standardTestCases/aliased_subworkflows/aliased_subworkflows.wdl" as subworkflow
+import "https://raw.githubusercontent.com/broadinstitute/cromwell/f2d1c3bdd5535d9f6a997eadaf136742d86adbe5/centaur/src/main/resources/standardTestCases/aliased_subworkflows/subworkflow.wdl" as subworkflow
```
Comment (mcovarr) [inline centaur/src/main/resources/standardTestCases/public_http_import/public_http_import.wdl]: 🤕

File: centaur/src/main/resources/standardTestCases/sub_workflow_decls/sub_workflow_decls.wdl (hunk 1 above)
Comment (cjllanwarne) [reply centaur/src/main/resources/standardTestCases/sub_workflow_decls/sub_workflow_decls.wdl]: The really sad thing is that the ‘input’ keyword at the top has not been treated to the en-reddening in the same way 😧

**Pull request merged**